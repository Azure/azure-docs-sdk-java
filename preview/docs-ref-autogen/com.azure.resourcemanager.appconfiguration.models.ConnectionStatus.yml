### YamlMime:JavaType
uid: "com.azure.resourcemanager.appconfiguration.models.ConnectionStatus"
fullName: "com.azure.resourcemanager.appconfiguration.models.ConnectionStatus"
name: "ConnectionStatus"
nameWithType: "ConnectionStatus"
summary: "Defines values for ConnectionStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class ConnectionStatus extends ExpandableStringEnum<ConnectionStatus>"
constructors:
- "com.azure.resourcemanager.appconfiguration.models.ConnectionStatus.ConnectionStatus()"
fields:
- "com.azure.resourcemanager.appconfiguration.models.ConnectionStatus.APPROVED"
- "com.azure.resourcemanager.appconfiguration.models.ConnectionStatus.DISCONNECTED"
- "com.azure.resourcemanager.appconfiguration.models.ConnectionStatus.PENDING"
- "com.azure.resourcemanager.appconfiguration.models.ConnectionStatus.REJECTED"
methods:
- "com.azure.resourcemanager.appconfiguration.models.ConnectionStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.appconfiguration.models.ConnectionStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.appconfiguration.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-appconfiguration:1.0.0-beta.1
