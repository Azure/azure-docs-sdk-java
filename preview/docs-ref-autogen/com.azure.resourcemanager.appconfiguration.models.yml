### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.appconfiguration.models"
fullName: "com.azure.resourcemanager.appconfiguration.models"
name: "com.azure.resourcemanager.appconfiguration.models"
summary: "Package containing the data models for AppConfigurationManagementClient. null."
classes:
- "com.azure.resourcemanager.appconfiguration.models.ActionsRequired"
- "com.azure.resourcemanager.appconfiguration.models.ApiKeyListResult"
- "com.azure.resourcemanager.appconfiguration.models.CheckNameAvailabilityParameters"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationResourceType"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStoreListResult"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStoreUpdateParameters"
- "com.azure.resourcemanager.appconfiguration.models.ConnectionStatus"
- "com.azure.resourcemanager.appconfiguration.models.EncryptionProperties"
- "com.azure.resourcemanager.appconfiguration.models.IdentityType"
- "com.azure.resourcemanager.appconfiguration.models.KeyVaultProperties"
- "com.azure.resourcemanager.appconfiguration.models.ListKeyValueParameters"
- "com.azure.resourcemanager.appconfiguration.models.OperationDefinitionDisplay"
- "com.azure.resourcemanager.appconfiguration.models.OperationDefinitionListResult"
- "com.azure.resourcemanager.appconfiguration.models.PrivateEndpoint"
- "com.azure.resourcemanager.appconfiguration.models.PrivateEndpointConnectionListResult"
- "com.azure.resourcemanager.appconfiguration.models.PrivateEndpointConnectionReference"
- "com.azure.resourcemanager.appconfiguration.models.PrivateLinkResourceListResult"
- "com.azure.resourcemanager.appconfiguration.models.PrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.appconfiguration.models.ProvisioningState"
- "com.azure.resourcemanager.appconfiguration.models.PublicNetworkAccess"
- "com.azure.resourcemanager.appconfiguration.models.RegenerateKeyParameters"
- "com.azure.resourcemanager.appconfiguration.models.ResourceIdentity"
- "com.azure.resourcemanager.appconfiguration.models.Sku"
- "com.azure.resourcemanager.appconfiguration.models.UserIdentity"
interfaces:
- "com.azure.resourcemanager.appconfiguration.models.ApiKey"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.Definition"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.DefinitionStages"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.DefinitionStages.Blank"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.DefinitionStages.WithEncryption"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.DefinitionStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.DefinitionStages.WithSku"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.DefinitionStages.WithTags"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.Update"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.UpdateStages"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.UpdateStages.WithEncryption"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.UpdateStages.WithIdentity"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.UpdateStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.UpdateStages.WithSku"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStore.UpdateStages.WithTags"
- "com.azure.resourcemanager.appconfiguration.models.ConfigurationStores"
- "com.azure.resourcemanager.appconfiguration.models.KeyValue"
- "com.azure.resourcemanager.appconfiguration.models.NameAvailabilityStatus"
- "com.azure.resourcemanager.appconfiguration.models.OperationDefinition"
- "com.azure.resourcemanager.appconfiguration.models.Operations"
- "com.azure.resourcemanager.appconfiguration.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.appconfiguration.models.PrivateEndpointConnection.Definition"
- "com.azure.resourcemanager.appconfiguration.models.PrivateEndpointConnection.DefinitionStages"
- "com.azure.resourcemanager.appconfiguration.models.PrivateEndpointConnection.DefinitionStages.Blank"
- "com.azure.resourcemanager.appconfiguration.models.PrivateEndpointConnection.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.appconfiguration.models.PrivateEndpointConnection.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.appconfiguration.models.PrivateEndpointConnection.DefinitionStages.WithPrivateEndpoint"
- "com.azure.resourcemanager.appconfiguration.models.PrivateEndpointConnection.DefinitionStages.WithPrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.appconfiguration.models.PrivateEndpointConnection.Update"
- "com.azure.resourcemanager.appconfiguration.models.PrivateEndpointConnection.UpdateStages"
- "com.azure.resourcemanager.appconfiguration.models.PrivateEndpointConnection.UpdateStages.WithPrivateEndpoint"
- "com.azure.resourcemanager.appconfiguration.models.PrivateEndpointConnection.UpdateStages.WithPrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.appconfiguration.models.PrivateEndpointConnections"
- "com.azure.resourcemanager.appconfiguration.models.PrivateLinkResource"
- "com.azure.resourcemanager.appconfiguration.models.PrivateLinkResources"
metadata: {}
package: "com.azure.resourcemanager.appconfiguration.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-appconfiguration:1.0.0-beta.1
