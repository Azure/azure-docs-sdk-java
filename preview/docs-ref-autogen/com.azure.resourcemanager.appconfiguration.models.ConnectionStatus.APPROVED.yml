### YamlMime:JavaMember
uid: "com.azure.resourcemanager.appconfiguration.models.ConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.appconfiguration.models.ConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "ConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.appconfiguration.models.ConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.appconfiguration.models.ConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "ConnectionStatus.APPROVED"
  summary: "Static value Approved for ConnectionStatus."
  syntax: "public static final ConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.appconfiguration.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-appconfiguration:1.0.0-beta.1
